starter-code/bulma.js

```diff
@@ -0,0 +1,54 @@
+Vue.component('message',{
```

Comment:
We prefer naming components in PascalCase and with at least two words. Here, it would look like `BulmaMessage`.